src/devices/tuya.ts

```diff
@@ -4232,6 +4391,131 @@ const definitions: Definition[] = [
             ],
         },
     },
+*/
+
+    {
+        // Since a lot of TuYa devices use the same modelID, but use different datapoints
+        // it's necessary to provide a fingerprint instead of a zigbeeModel
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_viy9ihs7']),
+
+        model: 'ZWT198 Avatto',
+        vendor: 'TuYa',
+        description: 'Avatto ZWT198 Wall Thermostat',
+        onEvent: tuya.onEvent({timeStart: '1970'}),
+
+        fromZigbee: [
+            tuya.fz.datapoints,
+            fzLocal.ZWT198_tuya_thermostat_schedule,
```

Comment:
Why is this needed? This should go via `tuya.fz.datapoints` instead

Reply:
datapoint dp109 results in 4 different states, due to splitting up the original schedule into two parts (workdays and holidays schedule, and two syntax check states, displaying possible syntax error for these schedules. I did not see a possibility to do this within` tuya.fz.datapoints`. If there is, I am open to it. Please advice!

Reply:
To return multiple properties at once, see https://github.com/Koenkk/zigbee-herdsman-converters/blob/3a10293dcd0563345b33849b3a45889312af0c3e/src/lib/tuya.ts#L515